### PR TITLE
API_LEVEL_19 and  software swipe support

### PR DESCRIPTION
**Pull request by abonnaudet-ledger**

### CODE REVIEW

File: speculos/mcu/screen.py [hunk 1]
```diff
@@ -171,20 +170,15 @@ def mousePressEvent(self, event: QMouseEvent):
         QApplication.setOverrideCursor(Qt.DragMoveCursor)
 
     def mouseReleaseEvent(self, event: QMouseEvent):
+        self.mouse_offset = event.pos()
         x, y = self._get_x_y()
         if x >= 0 and x < self._width and y >= 0 and y < self._height:
+            # Send a press to update the last pressed x,y to the app
```
Comment (xchapron-ledger) [inline speculos/mcu/screen.py:176]: Can you add a comment explaining that it is mainly to support the "software" swipe where a pressed event with an updated position must be sent before the released event.

**Review by xchapron-ledger — DISMISSED** [review]

**Comment by xchapron-ledger** [comment]
Would be nice to add swipe support in here: https://github.com/LedgerHQ/speculos/blob/master/speculos/api/finger.py

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [0.8.7] - 2024-??-??
+## [0.9.0] - 2024-05-06
+
+### Added
```
Comment (xchapron-ledger) [inline CHANGELOG.md:10]: + support for API_LEVEL_19

**Review by xchapron-ledger — COMMENTED** [review]
Has it been tested on at least boilerplate?

**Review by xchapron-ledger — APPROVED** [review]

**Comment by lpascal-ledger** [comment]
@xchapron-ledger swipe was tested and working on boilerplate. 

As for the Boilerplate tests suite, they fails on two types of errors:
- Incorrect snaphosts => wording changes ("swipe to continue" becomes "swipe to review")
- A Ragger bug following the changes in Speculos (`finger_touch(x, y, delay)` must be changed to `finger_touch(x, y, delay=delay)` else `delay` is interpreted as `x2`).

**Pull request merged**